doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -122,12 +128,11 @@ arise, don't panic. Common examples include:
 - An OSD in the Acting Set is ``down`` or unable to service requests, 
   and another OSD has temporarily assumed its duties.
 
-Ceph processes a client request using the **Up Set**, which is the set of OSDs
-that will actually handle the requests. In most cases, the Up Set and the Acting
-Set are virtually identical. When they are not, it may indicate that Ceph is
-migrating data, an OSD is recovering, or that there is a problem (i.e., Ceph
-usually echoes a "HEALTH WARN" state with a "stuck stale" message in such
-scenarios).
+In most cases, the Acting Set and the Up Set are virtually identical. When they
+are not, the most common case is a placement group movement from one OSD to
+another. But it may indicate an OSD is recovering from failure, or that there is
```

Comment:
@TheJJ I think

> When they are not, it may indicate that Ceph is migrating data, an OSD is recovering, ...

still applies. and might be more accurate. the reason why I don't like 

> But it may indicate an OSD is recovering from failure, or that there is a problem

is that, "failure" or "problem" is somehow a little bit misleading even in the context of a distributed system.

- we can have planned migration of cluster from on pool to another
- we can add more OSDs to the cluster
- we can upgrade / replace an fault drive of an OSD. i.e. replace an OSD with another.

none of the use case above should be identified as failure or problem, IMHO. so probably you could keep the existing wording?

Reply:
ok, thanks, i've updated it and nearly kept the original wording.